### PR TITLE
New Model: Urban Nature Access

### LINKED ISSUE #722: Implement Urban Nature Access model
This is a catch-all issue for implementing the Urban Nature Access model.

See https://github.com/natcap/softwareteam/issues/93 for details.

- [x] Add `ARGS_SPEC`
- [x] Implement validation
- [x] Add Qt interface based on `ARGS_SPEC`
- [x] Add sample data to the repository
- [x] Note model addition in `HISTORY.rst` 
- [x] Add assertions to the whole-model test
- [x] Add a note to the User's Guide about when fractional populations may be a good idea and when they may not, but also that the model will not do any rounding on population inputs.
- [x]  _more individual tasks coming soon_

### PR DESCRIPTION
**Pull request by phargogh**
## Description
Fixes #722 

## Checklist
- [X] Updated HISTORY.rst and link to any relevant issue (if these changes are user-facing)
- [X] Updated the user's guide (if needed)
- [X] Tested the affected models' UIs (if relevant)

### CODE REVIEW

File: src/natcap/invest/ui/urban_nature_access.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# coding=UTF-8
```
Comment (dcdenu4) [inline src/natcap/invest/ui/urban_nature_access.py:1]: Should we still add this over? I believe `ui/` has been pulled out already in the 3.13.0 release branch.

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Hey @phargogh, just one question about a now deprecated UI module.

File: src/natcap/invest/ui/urban_nature_access.py (hunk 1 above)
Comment (phargogh) [reply src/natcap/invest/ui/urban_nature_access.py:1]: Shoot, you're right.  I missed that.  I'll PR a thing back into the UNA branch.

Comment (phargogh) [reply src/natcap/invest/ui/urban_nature_access.py:1]: @dcdenu4 https://github.com/natcap/invest/pull/1248

**Review by dcdenu4 — APPROVED** [review]
Hoozah!!!

**Comment by phargogh** [comment]
🎉

**Pull request merged**